packages/elements/src/internals/machines/sign-in/verification.machine.ts

```diff
@@ -368,6 +369,9 @@ export const SignInFirstFactorMachine = SignInVerificationMachine.provide({
       const password = fields.get('password')?.value as string | undefined;
 
       switch (strategy) {
+        case 'passkey': {
+          return await parent.getSnapshot().context.clerk.client.signIn.authenticateWithPasskey();
+        }
```

Comment:
When "Submit" is hit we don't want to call attempt, instead call `authenticateWithPasskey`